editor/src/components/canvas/canvas-strategies/strategies/absolute-resize.test-utils.tsx

```diff
@@ -4,9 +4,14 @@ import { EdgePosition } from '../../canvas-types'
 import { mouseDragFromPointWithDelta } from '../../event-helpers.test-utils'
 import { EditorRenderResult } from '../../ui-jsx.test-utils'
 
+interface Point {
```

Comment:
Surely we already have a type like this?

Reply:
yes, but these are interfaces we don't want to expose to non-test code. in the editor we want to enforce the developer to pick CanvasPoint / WindowPoint etc. 

As far as I understand, it doesn't matter to Typescript (compile time or otherwise) if an interface is duplicated (but identical duck-typing wise) or if you are pointing to the same "reference" imported from the same declaration.

Reply:
Thinking out loud: The parameter is a delta and we could just make a `Delta` type as that is scope-agnostic.

Reply:
that's a good idea. I'm doing it!